autoload/cmake4vim.vim

```diff
@@ -176,7 +193,13 @@ endfunction
 
 " Functions allows to switch between build types
 function! cmake4vim#SelectBuildType(buildType) abort
-    let g:cmake_build_type = a:buildType
+    " so it's called only once
+    if ( !exists( 's:initialized_default_build_types' ) )
+        call cmake4vim#populateDefaultCMakeVariants()
+    endif
+    let s:initialized_default_build_types = v:true
+    let g:cmake_build_type = g:cmake_variants[ a:buildType ][ 'cmake_build_type' ]
```

Comment:
Maybe we can add private function `s:getCMakeVariants()` which should return up-to-date dictionary.